tests/src/ut_test_kvp.c

```diff
@@ -459,6 +459,25 @@ void test_ut_kvp_string(void)
     UT_ASSERT(status == UT_KVP_STATUS_SUCCESS );
     UT_ASSERT_STRING_EQUAL(result_kvp, "the beef is also dead" );
     UT_LOG( "checkStringDeadBeef2[%s]", result_kvp );
+
+}
+
+void test_ut_kvp_getFormattedFloatField( void )
+{
+    ut_kvp_status_t status;
+    char result_kvp[UT_KVP_MAX_ELEMENT_SIZE]={0xff};
+
+    UT_LOG_STEP("ut_kvp_getFormattedFloatField() - Check Float with Quotes for UT_KVP_STATUS_SUCCESS");
+    status = ut_kvp_getFormattedFloatField(gpMainTestInstance, "decodeTest.checkFloat", result_kvp, UT_KVP_MAX_ELEMENT_SIZE);
+    UT_ASSERT(status == UT_KVP_STATUS_SUCCESS );
+    UT_ASSERT_STRING_EQUAL(result_kvp, "5.1" );
```

Comment:
`UT_ASSERT_FLOAT_EQUAL` & `UT_ASSERT_DOUBLE_EQUAL` is required.

Reply:
have used UT_ASSERT for both double and float.